blog/source/hyper_vs_digitalocean_round_2_vpn_server.md

```diff
@@ -0,0 +1,83 @@
+title: "Hyper_ vs DigitalOcean Smackdown - Round 2: Set Up an OpenVPN Server on Hyper_"
+date: 2016-08-06 10:00:00 +0800
+author: hyper
+preview: A Virtual Private Network (VPN) is necessary when you want to traverse untrusted networks privately and securely to your Hyper_ cloud. In this tutorial, we'll set up an VPN server on a container in Hyper_ and then configure access to it from any place we like.
```

Comment:
an -> a?

configure access to it from any place we like -> configure it to access from any place we like.